tests/kind_config.yaml

```diff
@@ -0,0 +1,25 @@
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+  - role: control-plane
```

Comment:
here we set OIDC options for the control-plane node of the kind cluster the tests are executed in